requirements-dev.txt

```diff
@@ -2,7 +2,8 @@
 contexttimer
 coverage
 flake8
-mock
+importanize
```

Comment:
the only thing I want to track is mock since 1.1.x actually breaks one of the tests for some mysterious reason https://github.com/testing-cabal/mock/issues/283